pkg/op-brains/op_brains/chat/test_chat_model.py

```diff
@@ -1,12 +1,12 @@
-
 import groq
+import asyncio
 
 class GroqLLM:
-    def __init__(self, model_name='mixtral-8x7b-32768'):
+    def __init__(self, model_name='llama3-8b-8192'):
```

Comment:
would we keep this or use mixtral OR even openai?

Reply:
this class is just for testing purpose